Portfolio/index.html

```diff
@@ -1,177 +1,205 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <section id="Home"></section>
+    <h1>Doe.</h1>
+    <div class="navbar">
+        <ul>
+            <li>ABOUT</li>
+            <li>SKILLS</li>
+            <li>PROJECTS</li>
+            <li>REVIEWS</li>
+
+        </ul>
+        <img src="/phone.icon.png" alt="Css Img" width="20px" height="20px">
     </div>
 
-        <header>
-        <h1>About Me</h1>
-    </header>
-    <hr>
+    <div class="btn">
+        <button>
+            <bold>Contact Me</bold>

```

Comment:
The <div> tag should not be used inside a <h4> tag. Consider using <span> or <div> with a different class instead.